src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1836,6 +1936,26 @@ type PageInfo {
     endCursor: String
 }
 
+enum StreetMode {
+    """BICYCLE"""
+    BICYCLE
+
+    """CAR"""
+    CAR
+
+    """"Private car trips shared with others."""
```

Comment:
In our current implementation that is not a street mode. Carpool routes have a schedule and behave like transit.

Reply:
Ok, I'll remove it.